source/contracts/Augur.sol

```diff
@@ -3,13 +3,98 @@ pragma solidity 0.4.17;
+        return true;
+    }
+
+    function logWinningTokensRedeemed(IUniverse _universe, address _reporter, address _market, address _stakeToken, uint256 _amountRedeemed, uint256 _reportingFeesReceived, uint256[] _payoutNumerators) public returns (bool) {
+        require(_universe.isContainerForStakeToken(IStakeToken(msg.sender)));
+        WinningTokensRedeemed(_universe, _reporter, _market, _stakeToken, _amountRedeemed, _reportingFeesReceived, _payoutNumerators);
+        return true;
+    }
+
+    function logReportsDisputed(IUniverse _universe, address _disputer, address _market, uint8 _reportingPhase, uint256 _disputeBondAmount) public returns (bool) {
+        require(_universe.isContainerForMarket(IMarket(msg.sender)));
+        ReportsDisputed(_universe, _disputer, _market, _reportingPhase, _disputeBondAmount);
+        return true;
+    }
+
+    function logMarketFinalized(IUniverse _universe, address _market) public returns (bool) {
+        require(_universe.isContainerForMarket(IMarket(msg.sender)));
+        MarketFinalized(_universe, _market);
+        return true;
+    }
+
+    function logOrderCanceled(IUniverse _universe, address _shareToken, address _sender, bytes32 _orderId, uint8 _orderType, uint256 _tokenRefund, uint256 _sharesRefund) public onlyWhitelistedCallers returns (bool) {
+        OrderCanceled(_universe, _shareToken, _sender, _orderId, _orderType, _tokenRefund, _sharesRefund);
+        return true;
+    }
+
+    function logOrderCreated(IUniverse _universe, address _shareToken, address _creator, bytes32 _orderId, uint256 _price, uint256 _amount, uint256 _numTokensEscrowed, uint256 _numSharesEscrowed, uint256 _tradeGroupId) public onlyWhitelistedCallers returns (bool) {
+        OrderCreated(_universe, _shareToken, _creator, _orderId, _price, _amount, _numTokensEscrowed, _numSharesEscrowed, _tradeGroupId);
+        return true;
+    }
+
+    function logOrderFilled(IUniverse _universe, address _shareToken, address _creator, address _filler, uint256 _price, uint256 _numCreatorShares, uint256 _numCreatorTokens, uint256 _numFillerShares, uint256 _numFillerTokens, uint256 _settlementFees, uint256 _tradeGroupId) public onlyWhitelistedCallers returns (bool) {
+        OrderFilled(_universe, _shareToken, _creator, _filler, _price, _numCreatorShares, _numCreatorTokens, _numFillerShares, _numFillerTokens, _settlementFees, _tradeGroupId);
+        return true;
+    }
+
+    function logProceedsClaimed(IUniverse _universe, address _sender, address _market, uint256 _numShares, uint256 _numPayoutTokens, uint256 _finalTokenBalance) public onlyWhitelistedCallers returns (bool) {
+        ProceedsClaimed(_universe, _sender, _market, _numShares, _numPayoutTokens, _finalTokenBalance);
+        return true;
+    }
+
+    function logUniverseForked() public returns (bool) {
+        UniverseForked(msg.sender);
+    }
+
+    function logUniverseCreated(IUniverse _childUniverse) public returns (bool) {
```

Comment:
This should include parentUniverse as first parameter (that is the thing that needs to be trusted).

Reply:
Same as above, I now see that the parent universe is the caller, so it doesn't need to be a separate parameter.